src/ArduPilotPlugin.cc

```diff
@@ -285,20 +287,20 @@ class gz::sim::systems::ArduPilotPluginPrivate
   {
     // Extract data
     double range_max = _msg.range_max();
-    auto&& ranges = _msg.ranges();
-    auto&& intensities = _msg.intensities();
+    auto&& m_ranges = _msg.ranges();
```

Comment:
Gazebo has a convention that all member variables are camelCase with no `m_` or `_` prefix. They are then supposed to be accessed using this->memberVariable in code. Like it or loath it that's the Gazebo style (not a fan, but it's a carry over from the original version). We should stick to that to aid comparison with the built-in gz-sim plugins.